code/modules/mob/living/living.dm

```diff
@@ -1002,6 +1002,8 @@
 		return
 	if(SEND_SIGNAL(src, COMSIG_LIVING_TRY_PULL, AM, force) & COMSIG_LIVING_CANCEL_PULL)
```

Comment:
you have a perfectly good signal you could register onto here :) You'll also need to stop someone thats currently pulling and gets onto a skateboard.

Reply:
It is done